src/lib/libcore.js

```diff
@@ -2290,6 +2290,8 @@ addToLibrary({
   'element': 'anyfunc'
 });
 `,
+  $wasmTable__deps: ['$usedTableLength'],
+  $wasmTable__postset: `usedTableLength = wasmTable.length`,
```

Comment:
I don't think the `wasmTable` itself should depend on `usedTableLength`... it should only be folks that want to use `getEmptyTableSlot` that have this included.

Also, I think the `__postset` need to look more like `__postset: () => addAtInit(`usedTableLength = wasmTable.length`),`

I.e. that assignment of `usedTableLength` can only happen after the wasm module has been loaded.

Reply:
Ok, I introduced the dependencies, as the tests complained about not finding  `usedTableLength`, but it did not fix it.